data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
+	$(AM_V_GEN) $(SED)					\
+	-e "s|\@privileged_group\@|$(PRIVILEGED_GROUP)|"	\
+	-e "s|\@helper_user\@|$(EXT_APPS_HELPER_USER)|"		\
+	$< > $@
+
+polkit_rulesdir = $(datadir)/polkit-1/rules.d
+polkit_rules_DATA =					\
+	com.endlessm.SystemExternalApps.rules
+
+%.policy: %.policy.in
+	$(AM_V_GEN) $(MSGFMT) --xml -d $(top_srcdir)/po --template $< -o $@ || cp $< $@
```

Comment:
I don't think ever has any effect, because the file will always be overwritten by the sed rule below. I think you can probably just remove this line